mlflow/ml-package-versions.yml

```diff
@@ -69,64 +69,28 @@ tensorflow:
     minimum: "2.3.0"
     maximum: "2.9.1"
     requirements:
-      "< 2.6.0": ["protobuf<4.0.0"]
       # Requirements to run tests for keras
-      "== dev": ["scikit-learn", "pyspark", "pyarrow"]
+      ">= 0.0.0": ["scikit-learn", "pyspark", "pyarrow"]
     run: |
-      pytest tests/tensorflow/test_tensorflow2_model_export.py tests/tensorflow/test_tensorflow2_metric_value_conversion_utils.py
-
-      # Run tests for keras against keras-nightly that's installed with tf-nightly:
-      # https://github.com/tensorflow/tensorflow/blob/v2.6.0/tensorflow/tools/pip_package/setup.py#L110-L122
-      if [ "$PACKAGE_VERSION" == "dev" ]; then
-        pytest tests/keras/test_keras_model_export.py
-      fi
+      tf_version=$(pip freeze | grep 'tensorflow==' | grep -Eo '[0-9].[0-9]')
+      pip install tensorflow-estimator==${tf_version}
+      pytest \
+        tests/tensorflow/test_tensorflow_estimator_export.py \
+        tests/tensorflow/test_tensorflow2_core_model_export.py \
+        tests/tensorflow/test_tensorflow2_metric_value_conversion_utils.py \
+        tests/tensorflow/test_keras_model_export.py \
+        tests/tensorflow/test_keras_pyfunc_model_works_with_all_input_types.py
 
   autologging:
     minimum: "2.3.0"
     maximum: "2.9.1"
     requirements:
-      "< 2.6.0": ["protobuf<4.0.0"]
       "== dev": ["scikit-learn"]
     run: |
+      tf_version=$(pip freeze | grep 'tensorflow==' | grep -Eo '[0-9].[0-9]')
```

Comment:
This doesn't seem to work for tensorflow 2.10.0. See https://github.com/mlflow/mlflow/actions/runs/3061685547/jobs/4941774307.